api/v1alpha1/localization_types.go

```diff
@@ -13,13 +13,22 @@ import (
 	"k8s.io/apimachinery/pkg/types"
 )
 
+type Source struct {
+	// +optional
+	SourceRef *meta.NamespacedObjectKindReference `json:"sourceRef,omitempty"`
```

Comment:
Probably not; you're right. Most of our objects are namespaced.

Reply:
Ah, so. This is namespaced and kind because at the beginning the source could be either a snapshot or an action or a source. Since then I think we stayed with Snapshot mostly. But it could be any other CR.